framework/src/application/node/processor/processor.ts

```diff
@@ -441,6 +441,7 @@ export class Processor {
 			updateDelegates: async (delegates: Delegate[]): Promise<void> => {
 				await this._chain.setValidators(delegates, stateStore);
 			},
+			getDelegates: async (): Promise<Delegate[]> => this._chain.getValidators(),
```

Comment:
instead of `this._chain.getValidators` it should use `getValidators` function from `lisk-chain`, which takes the stateStore as input